core/modules/layout/layout.theme.inc

```diff
@@ -123,6 +123,14 @@ function theme_layout_settings_context_table($variables) {
 }
 
 /**
+ * Output the content for a custom text block.
+ */
+function theme_layout_custom_text_block($variables) {
+  $custom_text_block = $variables['element'];
+  return $custom_text_block;
+}
```

Comment:
This theme function doesn't seem necessary. Themers could target this block already by making a `block--layout--custom-text-block.tpl.php` file.